### PR TITLE
Remove connector form django demo

### PR DESCRIPTION
**Pull request by ZhmakaAS**
Remove `acra-connector` from django demo. Created separate Dockerfile to containerize django application with TLS. 
Deleted grafana acra-connector dashboard.

### CODE REVIEW

**Review by Lagovas — APPROVED** [review]
lgtm

**Review by shadinua — APPROVED** [review]

File: django/docker-compose.django.yml [hunk 1]
```diff
@@ -110,21 +69,30 @@ services:
...
             --db_port=5432
             --keys_dir=/keys
             --http_api_enable
+            --tls_auth=4
+            --tls_ca=/ssl/root.crt
+            --tls_cert=/ssl/acra-server.crt
+            --tls_key=/ssl/acra-server.key
+            --tls_client_id_from_cert
+            --tls_identifier_extractor_type=distinguished_name
+            --acraconnector_transport_encryption_disable
```
Comment (vixentael) [inline django/docker-compose.django.yml:95]: this flag should be here, right?

**Review by vixentael — APPROVED** [review]

File: django/docker-compose.django.yml (hunk 1 above)
Comment (Lagovas) [reply django/docker-compose.django.yml:95]: yes, it turns off acra-connector usage and waiting of trace/clientID info from connector side, and expects raw connection. in this case it is TLS connection

File: django/django.dockerfile [hunk 2]
```diff
@@ -0,0 +1,88 @@
+FROM alpine:3.9
```
[the remark this answers is not in the document]
Comment (shadinua) [reply django/django.dockerfile:1]: Gently reminder :) Looks like this wasn't updated.

**Pull request merged**